libraries/AP_GPS/AP_GPS.h

```diff
@@ -249,7 +249,8 @@ class AP_GPS
     void handle_msp(const MSP::msp_gps_data_message_t &pkt);
 #endif
 #if HAL_EXTERNAL_AHRS_ENABLED
-    void handle_external(const AP_ExternalAHRS::gps_data_message_t &pkt);
+    bool get_first_external_instance(uint8_t& instance) const WARN_IF_UNUSED;
```

Comment:
needs a comment to explain what this does

Reply:
Done.